z3-sys/src/lib.rs

```diff
@@ -7482,3 +7481,11 @@ extern "C" {
     /// Best-effort quantifier elimination
     pub fn Z3_qe_lite(c: Z3_context, vars: Z3_ast_vector, body: Z3_ast) -> Z3_ast;
 }
+
+#[cfg(not(windows))]
+#[link(name = "z3")]
+extern "C" {}
+
+#[cfg(windows)]
+#[link(name = "libz3")]
+extern "C" {}
```

Comment:
Are these still required even though we are printing the cargo linking commands? I would assume that they are not.

Reply:
Is that properly done for dynamic links too? The function where I changed that is guarded by `#[cfg(feature = "static-link-z3")]`

Reply:
Ah, good catch. You're correct.